activerecord/lib/active_record/relation.rb

```diff
@@ -255,6 +255,8 @@ def find_or_create_by!(attributes, &block)
     #   the problem of running out of integers, if the underlying table is still stuck on a primary
     #   key of type int (note: All \Rails apps since 5.1+ have defaulted to bigint, which is not liable
     #   to this problem).
+    # * You should not define uniqueness validator for columns that you defined database constraints for,
+    #   otherwise #create will fail with ActiveRecord::RecordInvalid and #find_by will never be called.
```

Comment:
We try to [avoid](https://edgeguides.rubyonrails.org/api_documentation_guidelines.html#pronouns) using "you" in the API docs.
Maybe something like:
```suggestion
    # * Columns with unique database constraints should not have uniqueness validations defined,
    #   otherwise #create will fail with ActiveRecord::RecordInvalid and #find_by will never be called.
```

Reply:
Thanks, it sounds better now.